examples/gitops/argocd-multi-cluster/hub-cluster/helm-grafana/values.yaml

```
@@ -0,0 +1,98 @@
+#Enabling IRSA module for extending the data sources to CloudWatch
+serviceAccount:
+  create: false
+
+service:
+  type: ${enable_ingress ? "ClusterIP" : "LoadBalancer"}
+
+ingress:
+  enabled: ${enable_ingress}
+  annotations:
+    alb.ingress.kubernetes.io/scheme: "internet-facing"
+    alb.ingress.kubernetes.io/target-type: "ip"
+    alb.ingress.kubernetes.io/listen-ports: "[{\"HTTPS\":443}]"
+    alb.ingress.kubernetes.io/tags: "Environment=hub,GitOps=true"
+  ingressClassName: "alb"
+  hosts:
+  - ${host}
+  tls:
+  - hosts: [${host}]
+
+resources:
+  limits:
+    cpu: 200m
+    memory: 512Mi
+  requests:
+    cpu: 200m
+    memory: 512Mi
+
+autoscaling:
+  enabled: true
+  minReplicas: 1
+  maxReplicas: 10
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      targetAverageUtilization: 60
+  - type: Resource
+    resource:
+      name: memory
+      targetAverageUtilization: 60
+
+nodeSelector:
+  kubernetes.io/os: ${operating_system}
+
+
+adminUser: admin
+# Set this password using set_sensitive values
+#adminPassword:
+
+persistence:
+  type: pvc
+  enabled: true
+  storageClassName: gp2
```

gp3 should be the default class for most things today
```suggestion
  storageClassName: gp3
```

---

although, that might require additional changes in terms of adding it as a storage class. you can disregard for now - future improvement